CODEOWNERS

```diff
@@ -1 +1 @@
-* @1debit/infrastructure-eng @1debit/security
+* @bwhaley @inga
```

Comment:
@bwhaley We could leave the previous owners to test if it works, otherwise if you have names to add here for now that'd be great.

Reply:
Can we make a new group in the new org called something like `@chime-maintainers` and add you, me, @arkadiyt , @Nicolas-Richard, @ejholmes to it, then update CODEOWNERS accordingly?